test/integration.js

```diff
@@ -130,7 +130,7 @@ describe('hoodie.store', function () {
       .then(done, done)
     }).then(toValue)
     .then(function () {
-      throw new Error('should reject with Conflict error')
+      Promise.reject(Error('should reject with Conflict error'))
```

Comment:
this is not correct, I’m afraid. The execution is not sync, and this code swallows the the rejected promise. You must add `return ` before the `Promise.reject(Error('should reject with Conflict error'))`, and then you will see the same errors again.

The problem is that either this code does not reject with a conflict error:

```js
hoodie.store.add({id: '456'})

.then(function () {
  return hoodie.store.add({id: '456'})
})
```

or that something is wrong with the `toValue` method. 

It seems to be the latter. When I run this code on https://tracker.hood.ie/, it logs `409` which is correct

```js
hoodie.store.add({id: '456'})
.then(function () {
  return hoodie.store.add({id: '456'})
})
.then(function () {
  console.log('Oooops, there should be a conflict error!')
})
.catch(function (error) {
  console.log(error.status)
})
```